fuel-vm/src/interpreter/executors/instruction.rs

```diff
@@ -75,62 +74,62 @@ where
 
         match op {
             OpcodeRepr::ADD => {
-                self.gas_charge(GAS_ADD)?;
+                self.gas_charge(self.gas_costs.add)?;
```

Comment:
Not familiar, but what are the performance costs of changing this from a constant which is inlined into an instance variable? Gas cost counting I know is significant on performance otherwise I wouldn't bring it up

Reply:
We have no choice if we want to be able to pass in the gas costs at runtime